.gitignore

```diff
@@ -1,4 +1,19 @@
-autom4te.cache/
+# Please do not use this ignore file to define platform specific files
+# e.g. auto generated files by IDE.
+#
+# For these purposes create a global .gitignore file, which is a list of rules
+# for ignoring files in every Git repository on your computer.
+#
+# https://help.github.com/articles/ignoring-files/#create-a-global-gitignore
+
+# The directories to ignore (do not add trailing '/'s, they skip symlinks).
+/autom4te.cache
+/m4
+/test_*
+/.build*
+/build
```

Comment:
Why these two? `/build` is a subdirectory that contains scripts (and shouldn't be ignored), and `/.build` is used by some scripts as the name of the build directory, but it's configurable.

Reply:
We use `/.build`, `/.build_*` and so on in our scripts. I'll remove `/build`